test/e2e-website/material-docs.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { test as base, expect } from '@playwright/test';
+import kebabCase from 'lodash/kebabCase';
+import FEATURE_TOGGLE from '../../docs/src/featureToggle';
+import { TestFixture } from './playwright.config';
```

Comment:
Is there a way to use global imports?
```suggestion
import FEATURE_TOGGLE from 'xxx/docs/src/featureToggle';
import { TestFixture } from './playwright.config';
```

Reply:
updated, it is possible!